terraform-environments/aws/terragrunt-dev/us-east-1/terragrunt-dev/050-github-aws-permissions/README.md

```diff
@@ -0,0 +1,57 @@
+# 050-github-aws-permissions
```

Comment:
# Applied manually

This has to be applied manually.  The reason is given in this `README.md` file.